src/main/java/com/google/sps/data/Post.java

```diff
@@ -152,9 +211,16 @@ public void requestToPost(HttpServletRequest request, String collegeId) {
       int postHour = Integer.parseInt(entity.getProperty("endHour").toString());
       int postMinute = Integer.parseInt(entity.getProperty("endMinute").toString());
       postTime.set(postYear, postMonth, postDay, postHour, postMinute);
-      
-      // If the post time is before the current time, delete from Datastore.
+ 
+      // If the post time is before the current time, delete the post from storage.
```

Comment:
How come we would add the post in the first place?

Reply:
I'm a bit confused as to why we're deleting from storage here.

Reply:
This occurs on the GET command. So a post is deleted if it is outdated.